Vidar/src/main/cpp/RobotContainer.cpp

```diff
@@ -45,6 +45,7 @@ RobotContainer::RobotContainer() : m_autonomousCommand(&m_subsystem) {
  frc::SmartDashboard::PutData("FlywheelShoot 0.3", new FlywheelShoot(mShooter, 0.3));
  frc::SmartDashboard::PutData("FlywheelShoot 0.5", new FlywheelShoot(mShooter, 0.5));
  frc::SmartDashboard::PutData("FlywheelShoot 0.9", new FlywheelShoot(mShooter, 0.9));
+ frc::SmartDashboard::PutData("FlywheelShoot 0.75", new FlywheelShoot(mShooter, 0.75));
```

Comment:
Why is FlywheelShoot being changed along with Feeder? 

If this change is useful, I would like to see this change in separate commit. Seeing changes for other systems lumped together confuses the story of adding Feeder,